reddit_decider/__init__.py

```diff
@@ -214,6 +214,62 @@ def get_variant(
     #     self, experiment_name: str, variant_name: str, **exposure_kwargs: Optional[Dict[str, Any]]
     # ) -> None:
 
+    def _get_dynamic_config_value(
+        self,
+        dynamic_config_name: str,
+        decider_func: Callable[[str, DeciderContext], any],
+    ) -> Optional[any]:
+        context_fields = self._decider_context.to_dict()
+        ctx = rust_decider.make_ctx(context_fields)
+        ctx_err = ctx.err()
+        if ctx_err is not None:
+            logger.warning(f"Encountered error in rust_decider.make_ctx(): {ctx_err}")
+
+        res = decider_func(dynamic_config_name, ctx)
+        if res is None:
+            return
+        error = res.err()
+        if error:
+            logger.warning(f"Encountered error in decider.get_value(): {error}")
+            return None
+        else:
+            return res.val()
+
+    def get_bool(self, dynamic_config_name: str) -> Optional[bool]:
```

Comment:
I thought [the api](https://docs.google.com/document/d/1oBasmDRTTHutlAAsok0gLv1kn-HsL0xIgmPNjLvH2YM/edit#) was this:
```
def get_bool(feature_name: str, default: bool = False) -> bool:
```
where we use `feature_name` for the lookup param and allow for specifying the `default` value instead of returning `None`?

Reply:
ah you are right. Let me update